bazel/copts.bzl

```diff
@@ -13,11 +13,15 @@ def copts():
             # We do not want to treat `syscall` warning as an error.
             # Future update of glog lib will fix this warning.
             "-Wno-error=deprecated-declarations",
+            "-std=c++17",
```

Comment:
I still don't understand why we need this here if we already have it in our `.bazelrc`, please add a thorough comment that explains why!